src/main/java/edu/kis/powp/jobs2d/command/OperateToCommand.java

```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class OperateToCommand implements DriverCommand{
+	 private Job2dDriver driver;
+	    private int x;
+	    private int y;
+	    
+	public OperateToCommand(Job2dDriver driver, int x, int y) {
```

Comment:
Fixed driver instance. Commands should be able to use the current driver (i.e., DriverManager.getCurrentDriver())